packages/core/application/core-application-browser-internal/integration_tests/utils.tsx

```diff
@@ -10,9 +10,9 @@ import React, { ReactElement } from 'react';
 import { act } from 'react-dom/test-utils';
 import { mount } from 'enzyme';
 import { I18nProvider } from '@kbn/i18n-react';
+import type { AppMountParameters } from '@kbn/core-application-browser';
 
-import { AppMountParameters } from '../types';
-import { MockedMounterTuple, Mountable } from '../test_types';
+import { MockedMounterTuple, Mountable } from '../src/test_helpers/test_types';
```

Comment:
I wasn't sure, should `{package}/src/test_helpers/` be moved to `{package}/test_helpers/` instead, WDYT?

Reply:
It doesn't really matter, as long as the package structure follows the naming conventions the tooling changes re going to assume.
The real question here is when do we want to tackle the changes to cores' package structures to conform to these now-enforced naming conventions, now or later?